src/lib/ui/thermostat/AccordionRow.tsx

```diff
@@ -0,0 +1,44 @@
+import { type PropsWithChildren, useState } from 'react'
+
+import { ChevronWideIcon } from 'lib/icons/ChevronWide.js'
+
+interface AccordionRowProps {
```

Comment:
@mikewuu Introduced this and I think we should use it as it makes the (possibly exported) type match the actual props: https://github.com/seamapi/react/pull/273

I think I forgot about this and missed it in a few other PRs. It's easy to patch up again later.

```suggestion
interface AccordionRowProps extends PropsWithChildren {
```